pipeline/src/main/java/edu/rosehulman/aixprize/pipeline/http/HttpConfigurationLoader.java

```diff
@@ -0,0 +1,82 @@
+package edu.rosehulman.aixprize.pipeline.http;
+
+import java.io.*;
+import java.util.*;
+
+import org.apache.commons.io.IOUtils;
+import org.json.*;
+
+public class HttpConfigurationLoader {
+	protected static final String CONFIG_FILE_NAME = "src/edu_rosehulman_aixprize/servers.json";
+
+	public static class NoConfigurationFound extends Exception {
+		private static final long serialVersionUID = -7148498252286690391L;
+
+		public NoConfigurationFound(Class<? extends HttpAnnotator> annotator) {
+			super("No configuration loaded for " + annotator.getName());
+		}
+	}
+
+
+	private static HttpConfigurationLoader self;
```

Comment:
Having just been looking at python, this is very disturbing to look at. (not a request for change)